tests/checkers/resource_files.py

```diff
@@ -61,3 +63,34 @@ def check_resource_file_name(file_path):
 def check_file_extensions(file, lines, options=None):
     """ Check that there is no file without extension. """
     yield 0, "the file does not have an extension"
+
+@sphinxlint.checker('')
+def check_is_resource_file_used(file, options=None):
```

Comment:
I'd move this check function above `check_file_extensions` to keep resource file checkers together.